lib/icinga/checkable-check.cpp

```diff
@@ -94,15 +94,17 @@ double Checkable::GetLastCheck() const
 	return schedule_end;
 }
 
-void Checkable::ProcessCheckResult(const CheckResult::Ptr& cr, const MessageOrigin::Ptr& origin)
+Checkable::ProcessingResult Checkable::ProcessCheckResult(const CheckResult::Ptr& cr, const MessageOrigin::Ptr& origin)
 {
+ 	using Result = Checkable::ProcessingResult;
```

Comment:
A space sneaked in here at the beginning of the line.

Reply:
Ouch, sorry about that